addons/website_event/views/event_templates_list.xml

```diff
@@ -99,47 +99,24 @@
                                 <div t-attf-class="o_wevent_offcanvas_cat_#{category.name} accordion-collapse collapse">
                                     <div class="accordion-body pt-0">
                                         <ul class="list-group list-group-flush">
-                                            <li t-if="category.is_published and category.tag_ids and any(tag.color for tag in category.tag_ids)" class="list-group-item">
-                                                <t t-foreach="category.tag_ids" t-as="tag">
-                                                    <a t-if="tag.color"
-                                                        t-att-href="'/event?%s' % keep_query('*', tags=str((search_tags - tag).ids if tag in search_tags else (tag | search_tags).ids))"
-                                                        t-attf-class="dropdown-item d-flex align-items-center justify-content-between #{'active' if tag in search_tags else ''}">
-                                                        <t t-out="tag.name"/>
+                                            <t t-foreach="category.tag_ids" t-as="tag">
+                                                <li t-if="category.is_published and category.tag_ids and any(tag.color for tag in category.tag_ids) and tag.color" class="list-group-item border-0 px-0">
```

Comment:
Indeed good fix, but should be/remain in its own commit

Can move most of the t-if before the t-foreach
```suggestion
                                            <t t-if="category.is_published and category.tag_ids and any(tag.color for tag in category.tag_ids)" t-foreach="category.tag_ids" t-as="tag">
                                                <li t-if="tag.color" class="list-group-item border-0 px-0">
```